demos/sample_azure_iot/sample_azure_iot.c

```diff
@@ -260,7 +261,7 @@ static void prvHandlePropertiesMessage( AzureIoTHubClientPropertiesResponse_t *
 
     LogInfo( ( "Property document payload : %.*s \r\n",
                pxMessage->ulPayloadLength,
-               pxMessage->pvMessagePayload ) );
+               (const char*)pxMessage->pvMessagePayload ) );
```

Comment:
nit : spaces like this:
 ( const char * ) pxMessage->pucCommandName

Reply:
same for all changes in this fil.e.

Reply:
Done.